platform-dom/src/main/java/com/softicar/platform/dom/input/auto/DomAutoCompleteList.java

```diff
@@ -13,6 +13,7 @@
 public class DomAutoCompleteList implements Iterable<String> {
```

Comment:
hmm, the constants deserve a comment, or better, make them both private and provide a static getter method with Javadoc, probably only for `MAXIMUM_ROW_COUNT`

Reply:
I find the distinction between ROW and ELEMENT quite esoteric, why not make the constant names speak for themselves? e.g. `MAXIMUM_ELEMENT_COUNT` -> `MAXIMUM_ELEMENTS_TO_DISPLAY` and `MAXIMUM_ROW_COUNT` -> `MAXIMUM_ELEMENTS_TO_LOAD`